plugin/adsk/scripts/mayaUsdTranslatorImport.mel

```diff
@@ -451,10 +451,21 @@ global proc string mayaUsdTranslatorImport_AppendFromDialog(string $currentOptio
     return $currentOptions;
 }
 
-global proc mayaUsdTranslatorImport_ViewHierCB()
+global proc mayaUsdTranslatorImport_ViewHierCB(string $parent)
 {
+    string $parentWindow = "";
+    {
+        string $result[];
+        string $windows = `window -parent $parent`;
```

Comment:
I'm unclear as to why you need to create a new window here to use as the parentWindow. Also is this window being destroyed?

Reply:
I misunderstood what the `window -parent` command does! I thought it queried the parent of the given window. I will investigate and fix it. (If you know the crrect way to query the parent of an existing window, it would be welcome.)

Reply:
I searched and don't see any Maya command that will return the parent of a window. One way I can think of would be to call python and use MQtUtil::findWindow() method to get the QWidget and then call parent() on that.

Reply:
I'll look around if there is a way to get the widget hierarchy from the panel name. Maybe something with `lsUI`.